### DISCUSSION TITLE
Klick'r 3.0.0-beta04

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#430] [Announcements]
> [!IMPORTANT]
> **This is a beta build.**
> It might be unstable so please backup your scenarios before updating to this version. It will be only availble on direct download here, and not on PlayStore/FDroid.

This release keep focusing on stabilizing the code for the public release. Several crashes have been fixed.
I have forgot to mention (and merge) a few utility features in the previous release note, so here they are!

## Menu Show/Hide

A new button is now available in the notification, the show/hide button, allowing to hide Klick'r menu while detecting.
![Capture d'écran 2024-06-07 183110](https://github.com/Nain57/Smart-AutoClicker/assets/8599025/cf038858-797c-44b3-b41e-0ebfa2adb8d7)

## Quick Settings Tile

You can now quickly load the last scenario runned with a quick settings tile. Check how to add new tiles and pick the Klick'r one!
![Capture d'écran 2024-06-07 183216](https://github.com/Nain57/Smart-AutoClicker/assets/8599025/28a5b26f-5821-49a8-931c-8d44e1a9578f)

**Full Changelog**: https://github.com/Nain57/Smart-AutoClicker/compare/3.0.0-beta03...3.0.0-beta04

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta04'>Klick'r 3.0.0-beta04</a>.</em>

**Comment by Mohmotte** [comment]
A setting "Hide when play scenario" should be better cause down button stop scenario ?

**Reply by Nain57** [reply]
This button is indeed problematic, because when the scenario starts, it is difficult to access to the notification.
I think a "Play and hide"/"Stop and show" will fit more use cases

**Comment by pretty-imp** [comment]
Hi! I like your superb app. Can you explain how to add a condition for swipe?
